### DISCUSSION TITLE
Lost all dishwasher entities after latest update

### DISCUSSION THREAD

**Discussion by imlovingithub** [ekutner/home-connect-hass#323] [Community Support] [answered]
Following the update to 1.0.2 I lost all the dishwasher entities/sensors.  I have an overnight automation set up to run the dishwasher which didn't run in the early hours of yesterday (2am to 5am).  Checking the entities for the dishwasher they all had the red warning icon but I stupidly forgot to make a note of what the warnings said (I think it was something along the lines of this sensor is no longer being provided?).

I removed home_connect_alt from HACS and reinstalled from scratch.  This time only one sensor (binary.sensor.nnnnnnnnnnnn_connected) has reappeared.  Previously there were 27 entities for this dishwasher.

I'm not sure what my next steps should be?  Any advice would be very much appreciated.

My HA is on OS 11.4 and core is on 2024.1.5

**Comment by imlovingithub** [comment]
Rolling back to version 1.0.0 the single remaining sensor also shows the red warning icon under status and the message is "This entity is no longer being provided by the home_connect_alt integration. If the entity is no longer in use, delete it in settings.".

**Comment by imlovingithub** [comment]
Debug is showing an error code 409...

2024-01-21 13:56:11.328 DEBUG (MainThread) [home_connect_async.api] API call failed with code=409 error=SDK.Error.HomeAppliance.Connection.Initialization.Failed
2024-01-21 13:56:11.328 DEBUG (MainThread) [home_connect_async.appliance] Failed to load selected programs with error code=409 key=SDK.Error.HomeAppliance.Connection.Initialization.Failed

**Comment by imlovingithub** [comment]
I have run debug in log mode 7 and attached it to an issue report

**Comment by imlovingithub** [comment, marked as answer]
Restarting the dishwasher fixed this in the end.

**Reply by Mr5Picker** [reply]
Thanks. This happened to me as well. Spent a few days trying to fix before finding this post. Needed to completely power off and on dishwasher to resolve.
